src/2982R1_std_quantity_as_a_numeric_type.md

```diff
@@ -2066,7 +2067,7 @@ static_assert(isq::height(2 * m) - isq::height(1 * m) == isq::height(1 * m));
 
 The above uses the same types for LHS, RHS, and the result, but in general, we can add, subtract,
 or compare the values of any quantity type as long as both quantities are of the same kind.
-The result of such an operation will be the common type of the arguments:
+The result of addition and subtraction will be the common type of the arguments:
```

Comment:
Reading this now, I suppose "such" refers to those in the example,
as might be implied by the colon at the end.
My original read included the "compare" of the previous sentence,
which returns a `bool` or comparison category type, and not a quantity type.